portality/templates/_js_includes.html

```diff
@@ -16,15 +16,18 @@
 <!-- get select2 -->
 <script type="text/javascript" src="/static/portality/vendor/select2-release-3.2/select2.min.js"></script>
 
-{% if search_page %}
+{% if search_page or subject_page %}
```

Comment:
We should go for some kind of declarative approach to JS dependencies in the future, all these if-s are getting a bit iffy, now nested I see :). (but not now in this PR)